src/Http/Controllers/Controller.php

```diff
@@ -168,7 +168,7 @@ public function insertUpdateData($request, $slug, $rows, $data)
                 $data->{$row->field} = str_replace($uuid, $data->getKey(), $data->{$row->field});
             });
             $data->save();
-            if ($old_path != $new_path && !Storage::disk(config('voyager.storage.disk'))->exists($new_path)) {
+            if ($old_path != $new_path && !Storage::disk(config('voyager.storage.disk'))->exists($new_path) && Storage::disk(config('voyager.storage.disk'))->exists($old_path)) {
```

Comment:
```suggestion
            $disk = Storage::disk(config('voyager.storage.disk'));
            if ($old_path != $new_path && !$disk->exists($new_path) && $disk->exists($old_path)) {
```

Just a readability change.  Can you also update the other usages of `Storage::disk(...)`?